azure-pipelines/build-pr.yml

```diff
@@ -206,7 +206,7 @@ jobs:
       - name: tag
         value: 'LungPR'
       - name: more_switches
-        value: '--num_epochs=1 --feature_channels=16 --show_patch_sampling=0 --train_batch_size=4 --perform_validation_and_test_set_inference=False'
+        value: '--num_epochs=1 --feature_channels=16 --show_patch_sampling=0 --train_batch_size=4'
```

Comment:
why was the inference flag removed?

Reply:
It's not supported any more, but I failed to replace it with the new ones...